pareto/utilities/get_data.py

```diff
@@ -236,9 +237,41 @@ def get_data(fname, set_list, parameter_list, sum_repeated_indexes=False):
     It is worth highlighting that the Set for time periods "model.s_T" is derived by the
     method based on the Parameter: CompletionsDemand which is indexed by T
     """
+    # Using only the names available in the input sheet
```

Comment:
Looks like the comments may be unfinished. I've added some suggestions, feel free to use or modify:
```suggestion
    # Check all names available in the input sheet
```